scripts/Game/index.js

```diff
@@ -643,7 +690,7 @@ function windowResized() {
 }
 
 function keyPressed() {
-  if (paused) {
+  if (paused || scene === "death") {
```

Comment:
If the scene = death, the game should already be paused, so you can get rid of the || scene === "death"